docs/content/tutorial-cmake.md

```diff
@@ -0,0 +1,146 @@
+CMake Tutorial {#tut_cmake}
+====
+\brief Generating LCM type bindings with CMake
+
+This tutorial will walk you through a sample `CMakeLists.txt` that will
+generate bindings for your LCM message types.
+
+First, of course, you will want to find LCM itself, and include it's "use file"
+(a "use file" is a CMake script provided with some packages that adds utility
+functions for using that package). This can be done in your root
+`CMakeLists.txt`. If you will be generating Python and/or Java bindings, now
+is a good time to find those also.
+
+    find_package(lcm REQUIRED)
+    include(${LCM_USE_FILE})
```

Comment:
Since only `${LCM_USE_FILE}` is used here, it's hard for people to find, inspect, and learn from the `lcmUtilities.cmake` file that it points to. I would mention the file name where 'use file' is mentioned first above.